src/router/index.ts

```diff
@@ -15,7 +15,7 @@ import { showToast } from '@/utils'
 import { translate } from '@/i18n'
 
 import 'vue-router'
-import { Login, useAuthStore } from '@hotwax/dxp-components';
+import { DxpLogin, useAuthStore } from '@hotwax/dxp-components';
```

Comment:
Upgrade dxp-component in package file, as current version of dxp is missing login component

Reply:
The latest release does not have the updated code.

Reply:
Updated with the new release.